xbmc/cores/VideoPlayer/DVDInputStreams/DVDInputStreamNavigator.cpp

```diff
@@ -540,10 +539,9 @@ int CDVDInputStreamNavigator::ProcessBlock(uint8_t* dest_buffer, int* read)
             free(times);
           }
         }
-        CLog::Log(LOGDEBUG, "{} - Cell change: Title {}, Chapter {}", __FUNCTION__, m_iTitle,
-                  m_iPart);
-        CLog::Log(LOGDEBUG, "{} - At position {:.0f}% inside the feature", __FUNCTION__,
-                  100 * (double)pos / (double)len);
+        CLog::LogF(LOGDEBUG, "Cell change: Title {}, Chapter {}", m_iTitle, m_iPart);
+        CLog::LogF(LOGDEBUG, "At position {:.0f}% inside the feature",
```

Comment:
Please update to the current log standards using `{}`.

For more information please see the [libfmt documentation](https://fmt.dev/latest/syntax.html)